docs/includes/features-lamindb.md

```diff
@@ -32,7 +32,7 @@ Manage storage (files, folders, arrays) and a SQL database (SQLite, Postgres) ba
 
 **Organize and share data across a mesh of LaminDB instances.**
 
-- Create & load database instances like git repos: `lamin init` & `lamin load`
+- Create & load database instances like git repos: {doc}`docs:cli#lamin-init` & {doc}`docs:cli#lamin-load`
```

Comment:
I think you need to add the file suffix: `cli.md`

You also don't need to prefix with `docs` here because you're in the master docs repo. But I agree it doesn't hurt.

Reply:
Well it might hurt because it leads to a more ugly preview pop up "in Lamin Docs" rather than just showing the link, but on that level, we have a 10 other things to do in these docs... so don't worry.